src/server.prod.js

```diff
@@ -40,18 +40,20 @@ class ProdServer {
 
   setupApp() {
     const app = express();
-    const serverPublicPath = this.config.server.output.publicPath;
-    const clientPublicPath = this.config.client.output.publicPath;
-    const serverOutputPath = this.config.server.output.path;
-    const clientOutputPath = this.config.client.output.path;
     const { before, after } = this.config.middleware;
     const nock = this.config.nock;
     const nockPath = this.config.nockPath;
     const accessLogs = this.config.accessLogs || 'clf';
     const renderFn = this.getRenderFunction();
+    if (!this.config.isCDN) {
```

Comment:
Maybe extract that into its own method? Like

```js
    if (!this.config.isCDN) this.setupStaticFiles(app);
```

?

Reply:
Done